extensions/sql-bindings/src/common/constants.ts

```diff
@@ -75,6 +75,10 @@ export const userPasswordLater = localize('userPasswordLater', 'In order to user
 export const openFile = localize('openFile', "Open File");
 export const closeButton = localize('closeButton', "Close");
 export const connectionProgressTitle = localize('connectionProgressTitle', "Testing SQL Server connection...");
+export const selectObjectName = localize('selectObjectName', 'Select object name method');
```

Comment:
I don't love this string. From figma:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/40371649/168936972-8ec267f1-9798-4d73-aaeb-2b82420f6fc0.png">

Since we only support tables, I'm not sure that we need to be super generic at the moment and say "Object", but that's minor. I'm ok leaving this.

Really what we're asking is whether they want to type in their own table name, or select a name from a list that we query on the user's behalf, right?

@dzsquared any chance that you could help here too with wording?

Reply:
Hey Chris, sorry for the confusion after few revisions I had not updated the figma as the experience. 

We removed the above in order to put the select table and enter manually together in one prompt shown here:
**Input Binding**
![image](https://user-images.githubusercontent.com/23587151/168986136-8fcec96f-3f00-4748-9686-632d5c9828c3.png)

**Output Binding**:
![image](https://user-images.githubusercontent.com/23587151/168986193-c6bb97d3-3816-430c-bf03-93cafbb7f746.png)

I have updated the figma as well to indicate the changes as well